usersync/cookie.go

```diff
@@ -30,9 +31,10 @@ var bidderToFamilyNames = map[openrtb_ext.BidderName]string{
 // To get an instance of this from a request, use ParsePBSCookieFromRequest.
 // To write an instance onto a response, use SetCookieOnResponse.
 type PBSCookie struct {
-	uids     map[string]uidWithExpiry
-	optOut   bool
-	birthday *time.Time
+	uids         map[string]uidWithExpiry
+	optOut       bool
+	birthday     *time.Time
+	maxSizeBytes int
```

Comment:
I don't think we need `maxSizeBytes` as part of the `PBSCookie` struct. We can just access the `MaxCookieSizeBytes` in the `HostCookie` struct and pass it on in the call to `SetCookieOnResponse`. This should make the code a little simpler. 

Spoke with @guscarreon offline about this and he said he will make the suggested change.

Reply:
done